codalab/server/bundle_manager.py

```diff
@@ -50,6 +52,8 @@ def __init__(self, codalab_manager):
 
         self._make_uuids_lock = threading.Lock()
         self._make_uuids = set()
+        # A set of UUID of bundles without their requested private worker discovered in the system
```

Comment:
"Set of bundle UUIDs with an unknown requested worker" (more general than just private worker, discovered in the system)